### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by gattytto**
Signed-off-by: Esteban Mañaricua <emanaricua@gmail.com>

NOTE: branch of flutter SDK is master because some stuff is only available in this branch (and the dev which still doesn't have milestone for required [fix](https://github.com/flutter/flutter/pull/62115)).

Signed-off-by: Esteban Mañaricua <emanaricua@gmail.com>

### CODE REVIEW

File: VERSION [hunk 1]
```diff
@@ -0,0 +1 @@
+master
```
Comment (ericwill) [inline VERSION]: This needs to be a plugin version.

File: VERSION [hunk 2]
```diff
@@ -0,0 +1 @@
+1.22.0-12.0
```
Comment (ericwill) [inline VERSION]: Is this the same version as what the plugin will be in the registry? The idea is that we want to keep the version field here and the plugin version the same.

Comment (gattytto) [reply VERSION]: it's the sdk version actually

**Comment by gattytto** [comment]
@ericwill it's now the same version as extension.

**Review by ericwill — APPROVED** [review]

File: VERSION (hunk 2 above)
Comment (benoitf) [reply VERSION]: @ericwill why the framework image needs to be the same than the extension version ?

if we have two VS Code extensions using flutter, they could reuse the same image

**Pull request merged**